docs/fastdds/ros2/ros2_configure.rst

```diff
@@ -155,21 +155,21 @@ XML configuration file location
 
 There are two possibilities for providing *Fast DDS* with XML configuration files:
 
-* **Recommended**: Setting the location with environment variable ``FASTRTPS_DEFAULT_PROFILES_FILE``
+* **Recommended**: Setting the location with environment variable ``FASTDDS_DEFAULT_PROFILES_FILE``
   to contain the path to the XML configuration file (see :ref:`env_vars`).
 
   ::
 
-      export FASTRTPS_DEFAULT_PROFILES_FILE=<path_to_xml_file>
+      export FASTDDS_DEFAULT_PROFILES_FILE=<path_to_xml_file>
 
 * **Alternative**: Placing the XML file in the running application directory
-  under the name *DEFAULT_FASTRTPS_PROFILES.xml*.
+  under the name *DEFAULT_FASTDDS_PROFILES.xml*.
 
 For example:
 
 .. code-block:: bash
 
-    export FASTRTPS_DEFAULT_PROFILES_FILE=<path_to_xml_file>
+    export FASTDDS_DEFAULT_PROFILES_FILE=<path_to_xml_file>
     export RMW_FASTRTPS_USE_QOS_FROM_XML=1
```

Comment:
Are we going to change also the RMW name?
If it applies after internal discussion, please apply here and following usages.

Reply:
No we are not going to change RMW for now. That is something we will discuss about in the future